src/main/java/gov/loc/repository/bagit/conformance/profile/BagInfoRequirement.java

```diff
@@ -10,7 +10,7 @@
 public class BagInfoRequirement {
   private boolean required;
   private List<String> acceptableValues = new ArrayList<>();
-  private boolean repeatable;
+  private boolean repeatable = true;
```

Comment:
We should probably add a comment to the constructor saying that repeatable is defaulted to true OR just remove the constructor that doesn't have repeatable as an argument.

Reply:
Ok